zappnode-front/src/landing/components/header/navigation.js

```diff
@@ -2,41 +2,60 @@ import React, {useState, useEffect} from 'react';
 import '../css/tailwind.generated.css';
```

Comment:
The css files only needs to be imported once at the root of the app. Like in App.js or Index.js

```suggestion

```

Reply:
okay, I will work on that